src/types/inc/User32Utils.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Routine Description:
+// - Retrieves the string resource from the current module with the given ID
+//   from the resources files. See resource.h and the .rc definitions for valid
+//   IDs.
+// Arguments:
+// - id - Resource ID
+// Return Value:
+// - String resource retrieved from that ID.
+inline std::wstring GetStringResource(const UINT id)

```

Comment:
Unless you truly want this inlined in every caller, you might need to strengthen this with `__declspec(noinline) inline` -- it means "emit one, AND ONLY ONE, copy of this function.. but do not inline it."